### PR TITLE
Replace "spaceflight" and "space travel" with "interstellar travel" where appropriate

### PR DESCRIPTION
**Pull request by abenkovskii**
I'm pretty sure you wanted to say "interstellar travel" instead of "spaceflight" there.

### CODE REVIEW

**Comment by tehhowch** [comment]
Spaceflight was discovered in the 1900s... as-is, Freedom says it was colonized in the next 82.3 years.

**Comment by Amazinite** [comment]
I'm pretty sure that "spaceflight" is shorthand for "invention of the hyperdrive" in a lot of places in the game, so you'll need to change a lot more than just the description of Freedom/

**Comment by abenkovskii** [comment]
$ find ./endless-sky/ -type f -exec cat '{}' + | grep -ci spaceflight 
  6

Only 6 occurrences of the word "spaceflight" in the source code of the game. I'll take a closer look at them later.

**Comment by abenkovskii** [comment]
I think this https://github.com/endless-sky/endless-sky/blob/master/data/quarg%20missions.txt#L16 can be left alone.

**Comment by abenkovskii** [comment]
$ find ./endless-sky/ -type f -exec cat '{}' + | grep -ci '\bspace.\?travel'
    5

5 occurrences of the phrase "space travel".

**Comment by abenkovskii** [comment]
Changing this two would just be nitpicking IMHO:

https://github.com/abenkovskii/endless-sky/blob/patch-1/data/transport%20missions.txt#L56
https://github.com/abenkovskii/endless-sky/blob/patch-1/data/ships.txt#L1533

And here "space travel" is actually correct:

https://github.com/abenkovskii/endless-sky/blob/patch-1/data/map.txt#L25496

**Comment by abenkovskii** [comment]
@Amazinite Done.

File: data/map.txt [hunk 1]
```diff
@@ -25443,7 +25443,7 @@ planet "Leviathan Station"
 	attributes deep station
 	landscape land/space2
 	music ambient/machinery
-	description `Leviathan station is the Deep's most productive deuterium harvesting station. Built nearly four hundred years ago, the station was one of the first to fuel the massive cargo caravans sent out from the Deep back when this was one of the only regions of the galaxy with a prosperous economy and a democratic government. Today it remains under government control, because the citizens of the Deep are wary of any private corporation controlling the industry that makes space travel possible.`
```
Comment (tehhowch) [inline data/map.txt]: Could go either way IMO

File: data/coalition missions.txt [hunk 2]
```diff
@@ -118,7 +118,7 @@ mission "Coalition: Contributor"
 		"reputation: Heliarch" += 10
 		set "license: Coalition"
 		log "Factions" "Saryds" `Saryds are an alien species who look suspiciously similar to the centaurs found in early human mythology. They prefer to live on worlds with as much green space as possible, and even their major cities are full of parks and gardens. Most Saryds live in small communes with up to a few dozen of them per house.`
-		log "Factions" "Kimek" `The Kimek are large insectoid aliens, and are members of the Coalition. They discovered spaceflight after the Saryds, but before the Arachi. They are intensely social creatures, preferring to live together in massive buildings with thousands of inhabitants, and many of their worlds have populations in the tens of billions, far more than any human world.`
```
Comment (tehhowch) [inline data/coalition missions.txt]: Could be fine as is.

File: data/remnant outfits.txt [hunk 3]
```diff
@@ -293,7 +293,7 @@ outfit "Forge-Class Thruster"
 	"flare sprite" "effect/remnant medium flare"
 		"frame rate" 4
 	"flare sound" "plasma small"
-	description "The Remnant has taken the basic ion thruster design that has been used by humanity since the dawn of spaceflight and made it significantly more powerful... but also more power-hungry."
```
Comment (tehhowch) [inline data/remnant outfits.txt]: Probably ok to not change, but as above, matter of opinion.

**Comment by endless-sky** [comment]
I think only the changes to the three human worlds make sense. For the Coalition, it makes perfect sense to talk about pre-spaceflight and post-spaceflight history. And for the ion engines, it actually *is* true that they've been used since the dawn of spaceflight. (They're commonly in use right now and have been for decades.)

**Comment by abenkovskii** [comment]
@endless-sky I stand corrected. I guess I got a bit overzealous. Reverted commits.

**Pull request merged**